pkg/distro/rhel/images.go

```diff
@@ -228,6 +228,7 @@ func osCustomizations(
 	var subscriptionStatus subscription.RHSMStatus
 	if options.Subscription != nil {
 		subscriptionStatus = subscription.RHSMConfigWithSubscription
+		osc.InsightsClientConfig = &osbuild.InsightsClientConfigStageOptions{Proxy: options.Subscription.Proxy}
```

Comment:
Should this also check if `options.Subscription.Proxy != ""`?  Otherwise we'll be adding a stage that sets the proxy to "", or it will generate a stage with empty options, which will probably fail.

Reply:
that's a good point, pushed that fix, thanks!

Reply:
Ty!